Xamarin.Forms.Platform.WPF/Renderers/EditorRenderer.cs

```diff
@@ -105,6 +108,14 @@ void UpdateTextColor()
 			Control.UpdateDependencyColor(System.Windows.Controls.Control.ForegroundProperty, Element.TextColor);
 		}
 
+		void UpdateMaxLength()
+		{
+			Control.MaxLength = Element.MaxLength;
```

Comment:
Not going to call anymore of these out, but there are quite a few more :)

Reply:
I think you got pretty much all of them? :P But thanks! Learning a lot here :)

Reply:
Yeah its fun, just doing these little optimizations on our first pass once we realized this was a problem gave Forms a fairly sizeable performance boost. Now its about making sure we dont end up needing to do that again :)